src/components/pages/collectRecordFormPages/BenthicLitForm/BenthicLitForm.js

```diff
@@ -70,18 +71,18 @@ const BenthicLitform = ({ isNewRecord }) => {
                 setIdsNotAssociatedWithData((previousState) => [...previousState, recordId])
               }
 
-              const recordNameForSubNode =
-                !isNewRecord && collectRecordResponse
-                  ? getRecordSubNavNodeInfo(
-                      collectRecordResponse.data,
-                      sitesResponse,
-                      collectRecordResponse.data.protocol,
-                    )
-                  : { name: language.protocolTitles.benthiclit }
+              setSubNavNode(
+                getDataForSubNavNode({
+                  isNewRecord,
+                  collectRecord: collectRecordResponse,
+                  sites: sitesResponse,
+                  protocol: 'benthiclit',
```

Comment:
you can also do collectRecordResponse.data.protocol instead of hardcoding this (makes it copy paste friendly and avoids hardcoding something that should ideally be in a constant/enum)